<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Emanuele Alfano</author>
 <class>SettingBoardWindow</class>
 <widget class="QDialog" name="SettingBoardWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SettingBoard</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>16</number>
    </property>
    <item>
     <widget class="QSplitter" name="EnSelect">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>12</number>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="EnMin">
        <item>
         <widget class="QLabel" name="valMinScr_2">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <strikeout>false</strikeout>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>En min [Step]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <property name="labelAlignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="formAlignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="en1Label_2">
            <property name="text">
             <string>En 1</string>
            </property>
            <property name="buddy">
             <cstring>en1Min</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="en1Min">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="minimum">
             <number>-99999</number>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="en2Label_2">
            <property name="text">
             <string>En 2</string>
            </property>
            <property name="buddy">
             <cstring>en2Min</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="en2Min">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="minimum">
             <number>-99999</number>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="en3Label_2">
            <property name="text">
             <string>En 3</string>
            </property>
            <property name="buddy">
             <cstring>en3Min</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="en3Min">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="minimum">
             <number>-99999</number>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="en4Label_2">
            <property name="text">
             <string>En 4</string>
            </property>
            <property name="buddy">
             <cstring>en4Min</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="en4Min">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="minimum">
             <number>-99999</number>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="en5Label_2">
            <property name="text">
             <string>En 5</string>
            </property>
            <property name="buddy">
             <cstring>en5Min</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="en5Min">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="minimum">
             <number>-99999</number>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="en6Label_2">
            <property name="text">
             <string>En 6</string>
            </property>
            <property name="buddy">
             <cstring>en6Min</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="en6Min">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="minimum">
             <number>-99999</number>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="EnMax">
        <item>
         <widget class="QLabel" name="valMinScr">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <strikeout>false</strikeout>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>En MAX [Step]</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <property name="labelAlignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="formAlignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="en1Label">
            <property name="text">
             <string>En 1</string>
            </property>
            <property name="buddy">
             <cstring>en1Max</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="en2Label">
            <property name="text">
             <string>En 2</string>
            </property>
            <property name="buddy">
             <cstring>en2Max</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="en2Max">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="minimum">
             <number>-99999</number>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="en3Label">
            <property name="text">
             <string>En 3</string>
            </property>
            <property name="buddy">
             <cstring>en3Max</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="en3Max">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="minimum">
             <number>-99999</number>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="en4Label">
            <property name="text">
             <string>En 4</string>
            </property>
            <property name="buddy">
             <cstring>en4Max</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="en4Max">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="minimum">
             <number>-99999</number>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="en5Label">
            <property name="text">
             <string>En 5</string>
            </property>
            <property name="buddy">
             <cstring>en5Max</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="en5Max">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="minimum">
             <number>-99999</number>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="en6Label">
            <property name="text">
             <string>En 6</string>
            </property>
            <property name="buddy">
             <cstring>en6Max</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="en6Max">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="minimum">
             <number>-99999</number>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="en1Max">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="minimum">
             <number>-99999</number>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="CurMax">
        <item>
         <widget class="QLabel" name="valMinScr_3">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <strikeout>false</strikeout>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>Cur Max [mA]</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_3">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <property name="labelAlignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="formAlignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="en1Label_3">
            <property name="text">
             <string>Mot1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="en2Label_3">
            <property name="text">
             <string>Mot2</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="en3Label_3">
            <property name="text">
             <string>Mot3</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="en4Label_3">
            <property name="text">
             <string>Mot4</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="en5Label_3">
            <property name="text">
             <string>Mot5</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="en6Label_3">
            <property name="text">
             <string>Mot6</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="curM1">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="minimum">
             <double>200.000000000000000</double>
            </property>
            <property name="maximum">
             <double>4000.000000000000000</double>
            </property>
            <property name="value">
             <double>2000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QDoubleSpinBox" name="curM6">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="minimum">
             <double>200.000000000000000</double>
            </property>
            <property name="maximum">
             <double>4000.000000000000000</double>
            </property>
            <property name="value">
             <double>2000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDoubleSpinBox" name="curM5">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="minimum">
             <double>200.000000000000000</double>
            </property>
            <property name="maximum">
             <double>4000.000000000000000</double>
            </property>
            <property name="value">
             <double>2000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="curM4">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="minimum">
             <double>200.000000000000000</double>
            </property>
            <property name="maximum">
             <double>4000.000000000000000</double>
            </property>
            <property name="value">
             <double>2000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="curM3">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="minimum">
             <double>200.000000000000000</double>
            </property>
            <property name="maximum">
             <double>4000.000000000000000</double>
            </property>
            <property name="value">
             <double>2000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="curM2">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="minimum">
             <double>200.000000000000000</double>
            </property>
            <property name="maximum">
             <double>4000.000000000000000</double>
            </property>
            <property name="value">
             <double>2000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>100</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>16</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="adcVref">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <strikeout>false</strikeout>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="statusTip">
             <string notr="true"/>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="midLineWidth">
             <number>0</number>
            </property>
            <property name="text">
             <string>Adc Vref</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="in1V1">
            <property name="text">
             <string>1.1V Interna</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="in2V56">
            <property name="text">
             <string>2.56V Interna</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ext">
            <property name="text">
             <string>Vref Esterna</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="adcOffset">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <strikeout>false</strikeout>
                <kerning>true</kerning>
               </font>
              </property>
              <property name="statusTip">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Offset Remove</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="offsetChose">
              <item>
               <property name="text">
                <string>Reading Directly</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Remove Offset</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="MotorPwm">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <strikeout>false</strikeout>
                <kerning>true</kerning>
               </font>
              </property>
              <property name="statusTip">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Motor PWM Freq</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="motFreqChose">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="editable">
               <bool>false</bool>
              </property>
              <property name="currentIndex">
               <number>3</number>
              </property>
              <item>
               <property name="text">
                <string>30 hz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>120 hz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>490 hz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4 Khz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>30 Khz</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QPushButton" name="saveSetting">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loadSetting">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="restoreSetting">
          <property name="text">
           <string>Restore</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="9"/>
 <tabstops>
  <tabstop>en1Min</tabstop>
  <tabstop>en1Max</tabstop>
  <tabstop>curM1</tabstop>
  <tabstop>en2Min</tabstop>
  <tabstop>en2Max</tabstop>
  <tabstop>curM2</tabstop>
  <tabstop>en3Min</tabstop>
  <tabstop>en3Max</tabstop>
  <tabstop>curM3</tabstop>
  <tabstop>en4Min</tabstop>
  <tabstop>en4Max</tabstop>
  <tabstop>curM4</tabstop>
  <tabstop>en5Min</tabstop>
  <tabstop>en5Max</tabstop>
  <tabstop>curM5</tabstop>
  <tabstop>en6Min</tabstop>
  <tabstop>en6Max</tabstop>
  <tabstop>curM6</tabstop>
  <tabstop>in1V1</tabstop>
  <tabstop>in2V56</tabstop>
  <tabstop>ext</tabstop>
  <tabstop>offsetChose</tabstop>
  <tabstop>motFreqChose</tabstop>
  <tabstop>saveSetting</tabstop>
  <tabstop>loadSetting</tabstop>
  <tabstop>restoreSetting</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingBoardWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingBoardWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
